src/components/modals/CheckoutProductModal.tsx

```diff
@@ -336,15 +337,23 @@ const CheckoutProductModal: React.VFC<CheckoutProductModalProps> = ({
       check.orderProducts[0].productId.includes('MerchandiseSpec_')
     ) {
       setIsCoinMerchandise(true)
+
+      let orderDiscountCoins = 0
+      check.orderDiscounts.map(orderDiscount => {
+        orderDiscountCoins += orderDiscount.options?.coins
+      })
```

Comment:
map expect return value, should be using forEach

Reply:
had fixed in below commit.
thx.